app/src/main/java/android/coding/interview/makeitawesome/fragment/PicturesFragment.java

```diff
@@ -25,13 +48,87 @@ public static Fragment newInstance() {
     @Nullable
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        RecyclerView rv = (RecyclerView) inflater.inflate(R.layout.pictures_list_fragment, container, false);
-        setupRecyclerView(rv);
-        return rv;
+        mPhotosAdapter = new PhotosAdapter(mData, getActivity());
+
+        //todo if there's no results show it
+        //tv_no_results = (TextView) findViewById(R.id.tv_no_results);
+
+        new HttpGetTask().execute(); //start background task
+
+
+        mRecyclerView = (RecyclerView) inflater.inflate(R.layout.pictures_list_fragment, container, false);
+        setupRecyclerView(mRecyclerView);
+        return mRecyclerView;
     }
 
     private void setupRecyclerView(RecyclerView recyclerView) {
+        mRecyclerView.addItemDecoration(new DividerItemDecoration(recyclerView.getContext(), DividerItemDecoration.VERTICAL_LIST));
```

Comment:
He took the time to search and reuse this DividerItemDecoration. Good point, I guess.